ztools/xssh/kex.go

```diff
@@ -165,7 +165,7 @@ type dhGroup struct {
 	JsonLog       dhGroupJsonLog
 }
 type dhGroupJsonLog struct {
-	Parameters      *ztoolsKeys.DHParams  `json:"params,omitempty"`
+	Parameters      *ztoolsKeys.DHParams  `json:"dh_params,omitempty"`
```

Comment:
It was pretty straightforward to make these changes to the existing code and get the output looking like what we want. I had some trouble completely wrapping my head around exactly which log structures are passed to which parts of the handshake, and further restructuring the logging to look like that suggestion would just take more time.